src/fullyautomatednutcracker/cogs/fun.py

```diff
@@ -8,6 +8,7 @@
 from utils import checks
 import asyncio
 import random
+import math
```

Comment:
whats this for?

Reply:
That's just what I accidentally added for something I'm working on locally